internal/cli/quickstarts.go

```diff
@@ -465,3 +344,54 @@ func defaultCallbackURLFor(s string) string {
 func defaultURL(url string, port int) string {
 	return fmt.Sprintf("%s:%d", url, port)
 }
+
+func (i *qsInputs) fromArgs(cmd *cobra.Command, args []string, cli *cli) error {
+	if !canPrompt(cmd) {
+		return errors.New("this command can only be run on interactive mode")
+	}
+
+	if len(args) == 0 {
+		if err := qsClientID.Pick(cmd, &i.ClientID, cli.appPickerOptions); err != nil {
+			return err
+		}
+	} else {
+		i.ClientID = args[0]
+	}
+
+	var client *management.Client
+	err := ansi.Waiting(func() (err error) {
+		client, err = cli.api.Client.Read(i.ClientID)
+		return
+	})
+	if err != nil {
+		return fmt.Errorf("an unexpected error occurred, please verify your client ID: %w", err)
+	}
+
+	i.Client = client
+	i.QsTypeForClient = quickstartsTypeFor(client.GetAppType())
+
+	quickstarts, err := auth0.GetQuickstarts()
```

Comment:
Should this be inside the spinner func?

Reply:
Good catch! 👍🏻  Definitely.

Reply:
[`c06a04f` (#553)](https://github.com/auth0/auth0-cli/pull/553/commits/c06a04f6d41f10032c2078fde1af74910bb0ec3e)